### PR TITLE
Add noticeboard default view.

### PR DESCRIPTION
**Pull request by maethu**
- The default view shows all categories, with their notices.
- The HTML-Structure is ready for a collapsible integration using jQuery or pure CSS.

Screen Shot 2020-05-15 at 10 53 51

### CODE REVIEW

File: ftw/noticeboard/browser/configure.zcml [hunk 1]
```diff
@@ -20,5 +20,13 @@
         class=".conditions.TermsAndConditions"
         />
 
+    <browser:page
+        for="ftw.noticeboard.content.noticeboard.INoticeBoard"
+        name="noticeboard_view"
```
Comment (busykoala) [inline ftw/noticeboard/browser/configure.zcml:25]: Do we have a preferred way of either using `-` or `_`? I noticed the previous routes in that pkg use `-`.

File: ftw/noticeboard/browser/templates/noticeboard.pt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<html xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      metal:use-macro="context/main_template/macros/master"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="ftw.noticeboard">
+
+    <div metal:fill-slot="content-core">
+        <tal:categories repeat="category view/get_categories_and_notices">
```
Comment (busykoala) [inline ftw/noticeboard/browser/templates/noticeboard.pt:8]: I think a class would help here to apply styles later on.
It is against YAGNI but can be very helpful. Maybe even wrap the whole stuff into its own container.

**Review by busykoala — APPROVED** [review]
Thanks!

File: ftw/noticeboard/browser/configure.zcml (hunk 1 above)
Comment (maethu) [reply ftw/noticeboard/browser/configure.zcml:25]: I usually prefer `-`, but it's kinda a convention in plone to use `_view` for content type default views.

File: ftw/noticeboard/browser/templates/noticeboard.pt (hunk 2 above)
Comment (maethu) [reply ftw/noticeboard/browser/templates/noticeboard.pt:8]: That's a good point. I'll add it once I apply some styles. It depends a little bit on the JS integration.

**Pull request merged**